test/extended/openstack/cpms.go

```diff
@@ -139,3 +169,14 @@ func getControlPlaneMachineSet(ctx context.Context, dc dynamic.Interface) (objx.
 	}
 	return cpmsList[0], nil
 }
+
+type FailureDomain struct {
```

Comment:
Please define this type locally to the test function, as it is not going to be useful elsewhere